packages/build/tests/mutate_save/fixtures/delete_headers/_headers

```diff
@@ -0,0 +1,2 @@
+/test
+  test: three
```

Comment:
Are these `mutate_save` fixtures used? Not seeing them on the tests and seem unrelated(?) with the PR?

Reply:
oh weird.. I didn't add these, unsure where they came from

Reply:
gonna rebase and see if they go away

Reply:
that didn't fix it so I just deleted them 🤷 we'll see